app.py

```diff
@@ -0,0 +1,1111 @@
+                            children=[
+                                html.Div(
+                                    id='panel-lower-indicators-0',
+                                    children=[
+                                        solar_panel_0,
+                                        thrusters,
+                                    ]
+                                ),
+                                html.Div(
+                                    id='panel-lower-indicators-1',
+                                    children=[
+                                        solar_panel_1,
+                                        motor,
+                                    ]
+                                ),
+                                html.Div(
+                                    id='panel-lower-indicators-2',
+                                    children=[
+                                        camera,
+                                        communication_signal
+                                    ]
+                                ),
+                            ]
+                        ),
+                        html.Div(
+                            id='panel-lower-graduated-bars',
+                            children=[
+                                fuel_indicator,
+                                battery_indicator
+                            ]
+                        ),
+                    ]
+                ),
+            ]
+        )
+    ],
+)
+
+##############################################################################################################
+# Data generation
+##############################################################################################################
+
+# Pandas
+df_non_gps_h = pd.read_csv('./data/non_gps_data_h.csv')
+df_non_gps_m = pd.read_csv('./data/non_gps_data_m.csv')
+df_gps_m = pd.read_csv('./data/gps_data_m.csv')
+df_gps_h = pd.read_csv('./data/gps_data_h.csv')
+
+# Satellite H45-K1 data
+df_non_gps_h_0 = pd.read_csv('./data/non_gps_data_h_0.csv')
+df_non_gps_m_0 = pd.read_csv('./data/non_gps_data_m_0.csv')
+df_gps_m_0 = pd.read_csv('./data/gps_data_m_0.csv')
+df_gps_h_0 = pd.read_csv('./data/gps_data_h_0.csv')
+
+# Satellite L12-5 data
+df_non_gps_h_1 = pd.read_csv('./data/non_gps_data_h_1.csv')
+df_non_gps_m_1 = pd.read_csv('./data/non_gps_data_m_1.csv')
+df_gps_m_1 = pd.read_csv('./data/gps_data_m_1.csv')
+df_gps_h_1 = pd.read_csv('./data/gps_data_h_1.csv')
+
+##############################################################################################################
+# Root
+##############################################################################################################
+root_layout = html.Div(
+    id='root',
+    children=[
+        dcc.Store(id='store-placeholder'),
+        dcc.Store(id='store-data', data={
+            'hour_data': {
+                'elevation': [df_non_gps_h['elevation'][i] for i in range(0, 60)],
```

Comment:
```suggestion
                'elevation': [df_non_gps_h['elevation'][i] for i in range(60)],
```

Pretty sure this has the same meaning, but correct me if I'm wrong!